netty/src/main/java/io/grpc/netty/NettyClientTransport.java

```diff
@@ -165,14 +166,20 @@ public ClientStream newStream(MethodDescriptor<?, ?> method, Metadata headers) {
   public Runnable start(Listener transportListener) {
     lifecycleManager = new ClientTransportLifecycleManager(
```

Comment:
I thought my comment explained that:
> Only use fields initialized before start()

Since `lifecycleManager` is initialized in `start()`, the flow in `start()` is more obvious when being passed as an argument. Otherwise you could move `newHandler()` within `start()` and not notice that a field is no longer initialized.